src/Domain/Entity/Event.php

```diff
@@ -283,4 +289,34 @@ public function getAttendees(): array
     {
         return $this->attendees;
     }
+
+    public function hasCategory(): bool
```

Comment:
This method should be called `hasCategories`.

```suggestion
    public function hasCategories(): bool
```

Reply:
I will change the method name and the push these changes.